kolibri/plugins/management/assets/src/device_management/deviceApp.js

```diff
@@ -22,6 +22,13 @@ const routes = [
     redirect: '/content',
   },
   {
+    path: '/welcome',
```

Comment:
I thought just tweaking the app's state rather than adding a completely different route just for the modal would be a little cleaner.